pcap/pcap.h

```diff
@@ -824,14 +824,6 @@ PCAP_API const char *pcap_lib_version(void);
 
   typedef struct pcap_send_queue pcap_send_queue;
 
-  /*!
```

Comment:
This should also not be removed, so that `pcap_get_airpcap_handle()` can be declared.